### PR TITLE
Utils guide

### PR DESCRIPTION
**Pull request by SantiagoDevRel**
Closing issue #6742

### CODE REVIEW

**Comment by luu-alex** [comment]
`Web3_Utils_module/Mastering_web3-utils.md` file and folder name should be lowercase

File: docs/docs/guides/Web3_Utils_module/Mastering_web3-utils.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+
+## Imports
+
+There are three different ways to import utils package.
+
+### Import the entire web3 library
+
+```js
+//import web3 module
+const { Web3 } = require("web3");
```
Comment (luu-alex) [inline docs/docs/guides/Web3_Utils_module/Mastering_web3-utils.md]:
```suggestion
import Web3 from "web3"; 
```
I suggest using typescript when creating examples, thats one of our strengths in v4!

File: docs/docs/guides/Web3_Utils_module/Mastering_web3-utils.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+toWei("")
+toHex("")
+```    
+
+## Methods example
+
+### Random Hex and Bytes
+
+```js
+//Random bytes in hex format and array format
```
Comment (luu-alex) [inline docs/docs/guides/Web3_Utils_module/Mastering_web3-utils.md]:
```suggestion
// Random bytes in hex format and array format
```
formatting, all comments using `//` should have a space before text

File: docs/docs/guides/Web3_Utils_module/Mastering_web3-utils.md [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+There are three different ways to import utils package.
+
+### Import the entire web3 library
+
+```js
+//import web3 module
+const { Web3 } = require("web3"); 
+
+//initialize  a provider
+const web3 = new Web3("https://eth.llamarpc.com");
```
Comment (luu-alex) [inline docs/docs/guides/Web3_Utils_module/Mastering_web3-utils.md]: we don't actually need to instantiate Web3, you can access web3 directly like `Web3.utils`. 

I think showing the user that u can access it through the way above would be great too^

File: docs/docs/guides/Web3_Utils_module/Mastering_web3-utils.md [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+console.log(web3.utils.compareBlockNumbers("pending", "latest"));
+//1
+
+console.log(web3.utils.compareBlockNumbers("latest", "pending"));
+//-1
+
+console.log(web3.utils.compareBlockNumbers("latest", "latest"));
+//0
+
+const equal2 = console.log(web3.utils.compareBlockNumbers(2, 2));
```
Comment (luu-alex) [inline docs/docs/guides/Web3_Utils_module/Mastering_web3-utils.md]:
```suggestion
console.log(web3.utils.compareBlockNumbers(2, 2));
```

File: docs/docs/guides/Web3_Utils_module/Mastering_web3-utils.md (hunk 1 above)
Comment (avkos) [reply docs/docs/guides/Web3_Utils_module/Mastering_web3-utils.md]: Do u mean 
`import { Web3 } from 'web3';`

File: docs/yarn.lock [hunk 5]
```diff
@@ -9066,4 +9066,4 @@ zwitch@^1.0.0:
 zwitch@^2.0.0:
   version "2.0.4"
   resolved "https://registry.yarnpkg.com/zwitch/-/zwitch-2.0.4.tgz#c827d4b0acb76fc3e685a4c6ec2902d51070e9d7"
-  integrity sha512-bXE4cR/kVZhKZX/RjPEflHaKVhUVl85noU3v6b8apfQEc1x4A+zBxjZ4lN8LqGd6WZ3dl98pY4o717VFmoPp+A==
+  integrity sha512-bXE4cR/kVZhKZX/RjPEflHaKVhUVl85noU3v6b8apfQEc1x4A+zBxjZ4lN8LqGd6WZ3dl98pY4o717VFmoPp+A==
```
Comment (avkos) [inline docs/yarn.lock]: yarn.lock shouldn't be changed. you do not have changes in package.json. please revert it

**Review by avkos — APPROVED** [review]
LGTM, except comments

**Pull request closed without merging**